pkg/agent/core/ngt/handler/grpc/handler_test.go

```diff
@@ -1176,69 +1191,1354 @@ func Test_server_Insert(t *testing.T) {
+				- case 2.2: Insert vector with different dimension (vector type is float32)
+				- case 3.1: Insert gaussian distributed vector success (vector type is uint8)
+				- case 3.2: Insert gaussian distributed vector success (vector type is float32)
+				- case 4.1: Insert uniform distributed vector success (vector type is uint8)
+				- case 4.2: Insert uniform distributed vector success (vector type is float32)
+
+		- Boundary Value Testing
+			- uint8, float32
+				- case 1.1: Insert vector with 0 value success (vector type is uint8)
+				- case 1.1: Insert vector with 0 value success (vector type is float32)
+				- case 2.1: Insert vector with min value success (vector type is uint8)
+				- case 2.2: Insert vector with min value success (vector type is float32)
+				- case 3.1: Insert vector with max value success (vector type is uint8)
+				- case 3.2: Insert vector with max value success (vector type is float32)
+				- case 4.1: Insert with empty UUID fail (vector type is uint8)
+				- case 4.2: Insert with empty UUID fail (vector type is float32)
+
+			- float32
+				- case 5: Insert vector with NaN value fail (vector type is float32)
+
+			- case 6: Insert nil insert request fail
+				* IncompatibleDimensionSize error will be returned.
+			- case 7: Insert nil vector fail
+				* IncompatibleDimensionSize error will be returned.
+			- case 8: Insert empty insert vector fail
+				* IncompatibleDimensionSize error will be returned.
+
+		- Decision Table Testing
+			- duplicated ID, duplicated vector, duplicated ID & vector
+				- case 1.1: Insert duplicated request fail when SkipStrictExistCheck is false (duplicated ID)
+					* AlreadyExists error will be returned.
+				- case 1.2: Insert duplicated request success when SkipStrictExistCheck is false (duplicated vector)
+				- case 1.3: Insert duplicated request fail when SkipStrictExistCheck is false (duplicated ID & vector)
+				- case 2.1: Insert duplicated request fail when SkipStrictExistCheck is true (duplicated ID)
+					* SkipStrictExistCheck flag is not used in agent handler, so the result is same as case 1.
+				- case 2.2: Insert duplicated request success when SkipStrictExistCheck is true (duplicated vector)
+				- case 2.3: Insert duplicated request fail when SkipStrictExistCheck is true (duplicated ID & vector)
+	*/
+	tests := []test{
+		// Equivalence Class Testing
+		func() test {
+			return test{
+				name: "Equivalence Class Testing case 1.1: Insert vector success (vector type is uint8)",
+				args: args{
+					ctx: ctx,
+					req: &payload.Insert_Request{
+						Vector: &payload.Object_Vector{
+							Id:     id,
+							Vector: intVec,
+						},
+					},
+				},
+				fields: fields{
+					name: name,
+					ip:   ip,
+					svcCfg: &config.NGT{
+						Dimension:    len(intVec),
+						DistanceType: ngt.Angle.String(),
+						ObjectType:   ngt.Uint8.String(),
+						KVSDB: &config.KVSDB{
+							Concurrency: 10,
```

Comment:
```suggestion
							Concurrency: 10,
```
Is it the right value in this test?
I suppose it is not...

Reply:
which value should we use?

Reply:
fixed by removing the default value here